assets/js/modules/analytics/dashboard/dashboard-widget-top-level.js

```diff
@@ -175,7 +175,7 @@ class AnalyticsDashboardWidgetTopLevel extends Component {
 						) }
 					{ ! permaLink && goals && isEmpty( goals.items ) && (
 						<CTA
-							title={ __( 'Use goals to measure success. ', 'google-site-kit' ) }
+							title={ __( 'Use goals to measure success.', 'google-site-kit' ) }
```

Comment:
Completely makes sense to remove trailing spaces, but we need to make sure that this does not somehow mess up spacing (in case those spaces were there for a reason). This also applies to the other cases where that was changed.

Reply:
This is the actual render of the following code:
`<h3 class="googlesitekit-cta__title">Use goals to measure success. </h3>`
![imagem](https://user-images.githubusercontent.com/7371591/68310263-9e15f900-00a7-11ea-963d-983e27e16a33.png)

The ending space isn't even used.

Reply:
For the rest of the strings where [I've removed the ending space](https://github.com/pedro-mendonca/site-kit-wp/commit/26c40e695d484a1673e8e5f4a1992f7dd2ffe0b8), here is my evaluation.

This is the actual render of the following code:
`<h3 class="googlesitekit-cta__title">Activate PageSpeed Insights. </h3>`
![imagem](https://user-images.githubusercontent.com/7371591/68314877-d5d46f00-00ae-11ea-98fa-6e59dcb6a4ef.png)

Again, the ending space is useless in the end of the `<h3>`.

Reply:
@felixarntz I can't think of a case where a trailing space in the source string would be necessary, can you?